README.md

```diff
@@ -110,74 +88,50 @@ with napari.gui_qt():
     # specify three sizes
     size = np.array([10, 20, 20])
     # add them to the viewer
-    points = viewer.add_points(points, size=size)
+    viewer.add_points(points, size=size)
 ```
 
 ![image](resources/screenshot-add-points.png)
 
-**napari** supports bidirectional communication between the viewer and the Python kernel, which is especially useful in Jupyter notebooks -- in the example above you can retrieve the locations of the points, including any additional ones you have drawn, by calling
+**napari** supports bidirectional communication between the viewer and the Python kernel, which is especially useful in Jupyter notebooks or from our built-in console. In this you can retrieve the locations of the points, including any additional ones you have drawn, by calling
```

Comment:
I think there should be a gif or annotated static image to show where to click to get the terminal.

Reply:
i have turned the screenshot below into a gif